projects/gateway2/translator/sslutils/ssl_utils.go

```diff
@@ -1,12 +1,31 @@
 package sslutils
 
 import (
+	"context"
 	"crypto/tls"
 	"fmt"
+	"strings"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/rotisserie/eris"
+	"github.com/solo-io/gloo/projects/gateway2/wellknown"
+	"github.com/solo-io/gloo/projects/gloo/pkg/api/v1/ssl"
+	"github.com/solo-io/go-utils/contextutils"
+	"google.golang.org/protobuf/types/known/wrapperspb"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/util/cert"
+	gwv1 "sigs.k8s.io/gateway-api/apis/v1"
+)
+
+// Gateway API has an extension point for implementation specific tls settings, they can be found [here](https://gateway-api.sigs.k8s.io/guides/tls/#extensions)
+const (
+	GatewaySslOptionsPrefix = wellknown.GatewayAnnotationPrefix + "/ssl"
+
+	GatewaySslCipherSuitesOption   = GatewaySslOptionsPrefix + "/cipher-suites"
+	GatewaySslMinimumTlsVersion    = GatewaySslOptionsPrefix + "/minimum-tls-version"
+	GatewaySslMaximumTlsVersion    = GatewaySslOptionsPrefix + "/maximum-tls-version"
```

Comment:
Would it make sense to nest these under `/parameters` to match the Edge API?

Reply:
Where else do we use `/parameters`?

Reply:
I mean that these options are under `sslConfig.parameters` so maybe we want to match the same path in this other format.